corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -232,6 +222,17 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
         },
     });
 
+    Views.CaseView = Views.CaseViewUnclickable.extend({
```

Comment:
Made the switch - not sure if there's a more elegant way to override parent functions with no-ops